pallets/parachain-staking/src/nomination_requests.rs

```diff
@@ -47,18 +47,18 @@ impl<Balance: Copy> NominationAction<Balance> {
 }
 
 /// Represents a scheduled request that define a [NominationAction]. The request is executable
-/// iff the provided [RoundIndex] is achieved.
+/// iff the provided [EraIndex] is achieved.
```

Comment:
existing typo?
```suggestion
/// if the provided [EraIndex] is achieved.
```

Reply:
Its not a typo, it means "if and only if"

Reply:
It's a standard in maths books

Reply:
But I do agree that it is ambiguous for anyone who does not have that background, and that we should be explicit and avoid abbreviations where possible